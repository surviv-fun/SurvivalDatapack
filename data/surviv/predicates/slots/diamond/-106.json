[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "offhand": {
                    "items": ["minecraft:diamond_sword", "minecraft:diamond_axe", "minecraft:diamond_pickaxe", "minecraft:diamond_shovel", "minecraft:diamond_hoe"]
                }
            }
        }
    }
]
