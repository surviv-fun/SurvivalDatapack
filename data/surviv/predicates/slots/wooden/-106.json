[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "offhand": {
                    "items": ["minecraft:wooden_sword", "minecraft:wooden_axe", "minecraft:wooden_pickaxe", "minecraft:wooden_shovel", "minecraft:wooden_hoe"]
                }
            }
        }
    }
]
