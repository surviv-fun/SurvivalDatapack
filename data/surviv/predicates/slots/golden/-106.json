[
    {
        "condition": "minecraft:entity_properties",
        "entity": "this",
        "predicate": {
            "equipment": {
                "offhand": {
                    "items": ["minecraft:golden_sword", "minecraft:golden_axe", "minecraft:golden_pickaxe", "minecraft:golden_shovel", "minecraft:golden_hoe"]
                }
            }
        }
    }
]
